src/platform/plugins/shared/dashboard/public/dashboard_actions/add_section_action.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { ADD_PANEL_ANNOTATION_GROUP } from '@kbn/embeddable-plugin/public';
+import { apiCanAddNewSection, CanAddNewSection } from '@kbn/presentation-containers';
+import { EmbeddableApiContext } from '@kbn/presentation-publishing';
+import { Action, IncompatibleActionError } from '@kbn/ui-actions-plugin/public';
+
+import { ACTION_ADD_SECTION } from './constants';
+
+type AddSectionActionApi = CanAddNewSection;
+
+const isApiCompatible = (api: unknown | null): api is AddSectionActionApi =>
+  Boolean(apiCanAddNewSection(api));
+
+export class AddSectionAction implements Action<EmbeddableApiContext> {
+  public readonly type = ACTION_ADD_SECTION;
+  public readonly id = ACTION_ADD_SECTION;
+  public order = 9;
```

Comment:
I would recommend bumping order. This seems like the most important of the group

Reply:
Good call - fixed in 719334fa5f3812fcd32ca85bf962ec69e0f40918

![image](https://github.com/user-attachments/assets/5a3a05c0-37a5-4d39-8bec-d75ee11f1c23)